CHANGES.rst

```diff
@@ -146,6 +147,7 @@ astropy.io.ascii
 
 astropy.io.misc
 ^^^^^^^^^^^^^^^
+- Added 'functional_models.py' to asdf/tags/transform. Added tags for Gaussian1D and Gaussian2D models to allow serialization. [#9959]
```

Comment:
Is this entry supposed to be here?

Reply:
no, i don't know why there are two astropy.io.misc sections in the 4.1 section of the change log. let me figure out which one to delete and just put this change in there.

Reply:
fixed that!